hipcheck/src/cli.rs

```diff
@@ -1181,6 +1201,29 @@ mod tests {
 		});
 	}
 
+	#[test]
+	fn resolve_policy_with_flag() {
+		let tempdir = TempDir::with_prefix(TEMPDIR_PREFIX).unwrap();
+
+		let expected = pathbuf![tempdir.path(), "HipcheckPolicy.KDL"];
```

Comment:
Nit: `KDL` ought to be lowercase

Reply:
I knew I would get that backwards somewhere. Fixed.